### PR TITLE
Few doc updates, mostly cleanups.

### PR DESCRIPTION
**Pull request by cijothomas**
Few doc updates for logs and some minor cleanups.

### CODE REVIEW

File: opentelemetry/src/lib.rs [hunk 1]
```diff
@@ -88,6 +88,16 @@
...
 //!
+//! 
+//! //! # Logs
+//!
+//!  The [`logs`] module contains the Logs Bridge API. It is not intended to be
+//!  called by application developers directly. It is provided for logging
+//!  library authors to build log appenders, that bridges existing logging
+//!  systems with OpenTelemetry. OpenTelemetry. Bridges for `log` and `tracing`
+//!  libraries are provided via the `opentelemetry-log` and
+//! `tracing-opentelemetry` crates.
```
Comment (TommyCpp) [inline opentelemetry/src/lib.rs:99]:
```suggestion
//!  library authors to build log appenders, that bridges existing logging
//!  systems with OpenTelemetry. Bridges for `log` and `tracing`
//!  libraries are provided via the `opentelemetry-log` and
//! `tracing-opentelemetry` crates.
```

File: opentelemetry/src/lib.rs [hunk 2]
```diff
@@ -88,6 +88,16 @@
...
 //! managing instruments.
 //!
+//! 
+//! //! # Logs
+//!
+//!  The [`logs`] module contains the Logs Bridge API. It is not intended to be
+//!  called by application developers directly. It is provided for logging
+//!  library authors to build log appenders, that bridges existing logging
+//!  systems with OpenTelemetry. OpenTelemetry. Bridges for `log` and `tracing`
+//!  libraries are provided via the `opentelemetry-log` and
```
Comment (lalitb) [inline opentelemetry/src/lib.rs]: s/opentelemetry-log/opentelemetry-appender-log/

File: opentelemetry/src/lib.rs [hunk 3]
```diff
@@ -88,6 +88,17 @@
 //! See the [`metrics`] module docs for more information on creating and
 //! managing instruments.
 //!
+//!
+//! //! # Logs
+//!
+//!  The [`logs`] module contains the Logs Bridge API. It is not intended to be
+//!  called by application developers directly. It is provided for logging
+//!  library authors to build log appenders, that bridges existing logging
+//!  systems with OpenTelemetry. OpenTelemetry. Bridges for
```
Comment (TommyCpp) [inline opentelemetry/src/lib.rs:97]:
```suggestion
//!  systems with OpenTelemetry. Bridges for
```
Hmmmm why do we have the second `Opentelemetry.` here

Comment (cijothomas) [reply opentelemetry/src/lib.rs]: Fixed!

**Review by TommyCpp — APPROVED** [review]

**Review by lalitb — APPROVED** [review]

**Pull request merged**